### DISCUSSION TITLE
can support Utility Groups?

### DISCUSSION THREAD

**Discussion by catch6** [tailwindlabs/tailwindcss#4197] [Ideas]
Windicss Utility Groups is very useful, will tailwindcss support it by jit?
```css
._box-right {
  @apply flex-1 pl-6 pr-6 pt-8 pb-16 bg-white sm:(w-full pl-8 pr-8) md:(w-1/2 pb-48);
}
```

**Comment by NikitaIT** [comment]
This duplicates for https://github.com/tailwindlabs/tailwindcss/discussions/4192
